rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/explorer/UnityExplorerFileSystemNode.kt

```diff
@@ -79,7 +79,7 @@ open class UnityExplorerFileSystemNode(project: Project,
         if (ignored || descendentOf == AncestorNodeType.IgnoredFolder) {
```

Comment:
Can we change `ignored` to `ignoredByIde`, please? It would help when understanding the logic of this - what's ignored by the IDE, and what's ignored by Unity

Reply:
Everywhere we use term HiddenAsset for Unity and Ignored for Rider. Doesn't require to be more specific.

Reply:
Sounds good to me